### PR TITLE
chore(docker)[OK-85] Reduce Docker images sizes

### PR DESCRIPTION
**Pull request by Wajktor13**
backend: `379MB -> 312MB`
frontend: `~1.4GB -> ~0.8GB`

### CODE REVIEW

File: frontend/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,45 @@
...
+FROM node:22-alpine3.19 AS run
+
+WORKDIR /app
+
+COPY package*.json ./
+
+COPY --from=build /app/node_modules /app/node_modules
+
+COPY --from=build /app/.next /app/.next
+
```
Comment (marcepanowyy) [inline frontend/Dockerfile:33]: Didn't you miss that?
```
COPY --from=build /app/next.config.mjs /app/next.config.mjs
```

File: backend/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
-RUN npm install
+RUN npm install --omit dev
+
+# Stage 2: build
+FROM node:22-alpine3.19 AS build
+
+WORKDIR /app
+
+COPY package*.json ./
+
```
Comment (marcepanowyy) [inline backend/Dockerfile]: I would reduce spacing between COPY and ARG, delete comments to make it more readable, e.g. :
```
COPY package*.json ./
COPY tsconfig.json ./
COPY --from=deps /app/node_modules /app/node_modules
COPY . .
```

**Review by marcepanowyy — APPROVED** [review]
Nice one

File: frontend/Dockerfile (hunk 1 above)
Comment (Wajktor13) [reply frontend/Dockerfile:33]: No, it is copied and used at the build stage.

**Review by Strattoss — APPROVED** [review]
All works fine and as expected

**Review by adbista — APPROVED** [review]

**Pull request merged**